src/db.ts

```diff
@@ -11,8 +11,7 @@ import type {
   QueryConfig,
   QueryResult,
 } from 'pg';
-// This needs to be imported as `*`, otherwise it will fail in esm
-import * as pg from 'pg';
```

Comment:
**question (if-minor):** there is an explicit comment that esm would fail 🤔 
you sure that this is correct? why would it suddenly work?

Reply:
Run `node bin/node-pg-migrate.mjs up` (**mjs**) and you will get `TypeError: pg.Client is not a constructor` instead of the connection error you receive with `node bin/node-pg-migrate.js up` (**js**).

With `import pg from 'pg'` you will only see the connection error in both cases, which is as expected

Reply:
I will look into this again later.
Maybe this should be extracted into its own PR.